core/src/main/kotlin/org/evomaster/core/search/service/Archive.kt

```diff
@@ -398,4 +406,13 @@ class Archive<T> where T : Individual {
 
         return current[0].fitness.doesCover(target)
     }
+
+    //update: support to save archive at some time point
```

Comment:
do not use "update", but rather TODO or FIXME, as those are recognized by the IDE

Reply:
fixed